shared/pkg/backendclient/client.go

```diff
@@ -619,3 +619,34 @@ func (b *BackendClient) PostFinding(ctx context.Context, finding models.Finding)
 		return nil, fmt.Errorf("failed to create a finding. status code=%v", resp.StatusCode())
 	}
 }
+
```

Comment:
This bloats the `backendclient` unnecessarily https://github.com/openclarity/vmclarity/pull/308/files#diff-189663ef212b0528411bdd1bc250e2c49471f3ae97cf265d63bf6d087212b314R623-R652
(btw, do not get having the `GetScanResultSummary` either)